projects/packages/stats/src/class-wpcom-stats.php

```diff
@@ -305,6 +305,45 @@ public function get_visits( $args = array() ) {
 		return $this->fetch_stats( $args );
 	}
 
+	/**
+	 * Get streaks for the site.
+	 *
```

Comment:
Thank you so much for extending our Stats package and adding tests!

Really appreciate it ❤️ 

Could you please also add the links to the corresponding WPCOM endpoints, same with the rest existing methods and also update the README.
This will greatly facilitate future consumers of this package!